docs/predictions.md

```diff
@@ -0,0 +1,55 @@
+# Making Predictions
+
+Healthcareai provides a few options when you want to get predictions from a trained model. Specifically these predictions come from an instance of TrainedSupervisedModel.
+
+Please note that you will likely only need one of these prediction output types.
```

Comment:
Maybe switch the order of lines 5 and 7? I was confused at first when I read line 5 before I read line 7.